server/lib/tuist/qa/agent.ex

```diff
@@ -39,15 +49,45 @@ defmodule Tuist.QA.Agent do
              auth_token: auth_token,
              account_handle: account_handle,
              project_handle: project_handle
+           }),
+         {:ok, log_streamer} <-
+           Client.start_log_stream(%{
+             server_url: server_url,
+             run_id: run_id,
+             auth_token: auth_token
            }) do
```

Comment:
Once we extract the agent into an Elixir app that runs as part of an executable. Does it make sense for this one to be added to the application's tree in `application.ex`?